setup.py

```diff
@@ -5,31 +5,44 @@
 from setuptools import setup
 from setuptools.command.install import install
 
-FULLVERSION = '0.0.2-2'
+FULLVERSION = "0.0.3"
```

Comment:
I see this is hard-coded here. So does it have to be updated each time? Or where should we modify the version number?

Reply:
Yes, and there's no apparent way to fix this. It's here and in the GitHub release part. Everything else is propagated from them.